docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -7,25 +7,37 @@ description: "Set up the Camunda 8 environment with Helm and an optional DNS set
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-This guide offers a comprehensive guide for installing the Camunda 8 Helm chart on your pre-existing AWS Kubernetes EKS cluster. Additionally, it includes instructions for setting up an optional DNS configuration.
+This guide provides a comprehensive walkthrough for installing the Camunda 8 Helm chart on your existing AWS Kubernetes EKS cluster. It also includes instructions for setting up optional DNS configurations and other optional AWS-managed services, such as OpenSearch and PostgreSQL.
+
 Lastly you'll verify that the connection to your Self-Managed Camunda 8 environment is working.
 
-## Prerequisites
+## Introduction
```

Comment:
This can likely just be removed given there isn't any content, and the headers can be reduced by one #.